backend/geneaprove/views/to_json.py

```diff
@@ -184,5 +184,5 @@ def post(self, request, *args, **kwargs):
             params.set_from_body(request.body)
         else:
             params.set_files(request.FILES)
-        logger.debug('%s.%s.post(%s)', self.__module__, self.__class__.__name__, params)
+        # logger.debug(f'{self.__module__}.{self.__class__.__name__}.post({params})')
```

Comment:
likewise

Reply:
See above message. I'd commented them during debugging and not uncommented for my commit. Both corrected and f-stringified.